AdminUi/test/AdminUi.Tests.Integration/AdminUi.Tests.Integration.csproj

```diff
@@ -40,4 +40,8 @@
     </None>
   </ItemGroup>
 
+  <ItemGroup>
+    <Folder Include="Features\IndividualQuotas\" />
+  </ItemGroup>
```

Comment:
Can be removed (?)

Reply:
Yes, not sure why sometimes this gets added automatically.